### PR TITLE
raidboss/oopsy: add Everkeep (normal) package

### PR DESCRIPTION
**Pull request by wexxlee**
Tested in emulator and in game.

This re-uses a lot of the tile calculation logic I did for EverkeepEx, but determining/calling safe spots is handled a little differently because the forged track patterns are both fixed and simpler.  For calling Forged Track, it will call a single inner safe tile regardless of how many tiles are actually safe, with priority given to the inner N/S tiles.  I *think* this probably fine, but given the recent discussion on #429 & #431, I wanted to mention that.  If folks prefer a different priority order, feel free to PR a config option, but I think this is fine for an initial pass at least.

### CODE REVIEW

**Comment by jacob-keller** [comment]
I think for normal mode and simplicity this is fine! Hopefully I can give it a whirl soon.

File: ui/raidboss/data/07-dt/trial/zoraal-ja.txt [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+##### PHASE 1 #####
+
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+8.8 "--sync--" StartsUsing { id: "934B", source: "Zoraal Ja" }
+13.8 "Soul Overflow" Ability { id: "934B", source: "Zoraal Ja" }
+21.1 "--center--" Ability { id: "98E1", source: "Zoraal Ja" }
+27.4 "Double-edged Swords (first)" Ability { id: "9352", source: "Zoraal Ja" } window 5,1
+29.7 "Double-edged Swords (second)" #Ability { id: "9352", source: "Zoraal Ja" }
+37.9 "Patricidal Pique" Ability { id: "9353", source: "Zoraal Ja" }
+41.1 "--sync--" Ability { id: "98E1", source: "Zoraal Ja" }
```
Comment (JLGarber) [inline ui/raidboss/data/07-dt/trial/zoraal-ja.txt:20]: Here and elsewhere, not all 98E1 abilities got marked as `--center--`

File: ui/raidboss/data/07-dt/trial/zoraal-ja.txt [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+293.0 "--sync--" StartsUsing { id: "9370", source: "Zoraal Ja" } window 293,1
+300.0 "Soul Overflow" Ability { id: "9370", source: "Zoraal Ja" }
+300.0 "--untargetable--"
+381.8 "--targetable--"
+
+##### PHASE 2 #####
+# Use NpcYell to detect the start of Phase 2 (since fight can reset to here)
+# Zoraal Ja: "This power is proof of my authority!"
+391.3 "--sync--" NpcYell { npcNameId: "2493", npcYellId: "44BF" } window 391.3,20
+399.0 "--sync--" Ability { id: "9355", source: "Zoraal Ja" }
```
Comment (JLGarber) [inline ui/raidboss/data/07-dt/trial/zoraal-ja.txt:61]: Same for this, all instances of 9355 should be marked `--center--`.

File: ui/raidboss/data/07-dt/trial/zoraal-ja.txt [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+# 934C Calamity's Edge
+# 934D Burst
+# 934E Vorpal Trail
+# 934F Vorpal Trail
+# 9350 Vorpal Trail
+# 9351 Double-edged Swords
+# 9352 Double-edged Swords
+# 9353 Patricidal Pique
+# 9354 Dawn of an Age
+# 9355 --sync--
```
Comment (JLGarber) [inline ui/raidboss/data/07-dt/trial/zoraal-ja.txt:153]:
```suggestion
# 9355 --sync-- (re-center)
```

File: ui/raidboss/data/07-dt/trial/zoraal-ja.txt [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+# 9374 Duty's Edge
+# 9375 Duty's Edge
+# 9376 Duty's Edge
+# 9378 Fire III
+# 9379 Bitter Reaping
+# 937A Bitter Reaping
+# 93A6 攻撃
+# 9527 Vorpal Trail
+# 9528 Vorpal Trail
+# 98E1 --sync--
```
Comment (JLGarber) [inline ui/raidboss/data/07-dt/trial/zoraal-ja.txt:188]:
```suggestion
# 98E1 --sync-- (re-center)
```

**Review by JLGarber — APPROVED** [review]
Thanks for putting all that together. Sorry I haven't been able to review stuff much these last couple of weeks, life has been quietly busy. I left a couple minor change requests, but go ahead and merge once you take care of those.

File: ui/raidboss/data/07-dt/trial/zoraal-ja.txt (hunk 1 above)
Comment (wexxlee) [reply ui/raidboss/data/07-dt/trial/zoraal-ja.txt:20]: Thanks for catching!  Will update this before I merge. 👍

**Pull request merged**